app.json

```diff
@@ -2,9 +2,20 @@
   "name": "api.mozillascience.org",
   "description": "API server for science.mozilla.org",
   "addons": [
-    "heroku-postgresql"
+    "heroku-postgresql:hobby-dev"
```

Comment:
As I recall from another project, hobby-dev databases have no backup running on them. How important is it that this data is not "just gone" when someone accidentally wipes the db? /cc @cadecairos